packages/yew/src/virtual_dom/mod.rs

```diff
@@ -57,6 +58,7 @@ pub enum AttrValue {
 impl Deref for AttrValue {
     type Target = str;
 
+    #[inline(always)]
```

Comment:
I don't think theses `inline` annotations actually effect anything, might as well remove them again.

Reply:
I think we should just keep them as they hint the compiler. The stdlib also has them